packages/api-explorer/__tests__/Response.test.jsx

```diff
@@ -74,5 +74,6 @@ test('should show different component tabs based on state', () => {
   expect(doc.html().includes('Response Headers')).toBe(true);
```

Comment:
We could stop Response Test with the check of ResponseBody prop and ResponseMetadata.
In ResponseMetadata's test we check RequestData, ResponseHeaders, ecc...

Reply:
I'd not change the original tests if not strictly required